scripts/generate-clientset.sh

```diff
@@ -32,91 +32,25 @@
 # Alternatively, make a symlink from the GOPATH package source directory to the
 # repo directory.
 
-set -euox pipefail
+set -euo pipefail
 
-GOPATH="${GOPATH:-$(go env GOPATH)}"
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+CODEGEN_PKG=${CODEGEN_PKG:-$(cd "${SCRIPT_ROOT}"; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo ../code-generator)}
```

Comment:
Do we have `code-generator` in the root directory?

Reply:
It's [vendored[(https://github.com/GoogleContainerTools/kpt-config-sync/tree/main/vendor/k8s.io/code-generator)

Reply:
I meant the fallback path `echo ../code-generator`, which seems to reference a `code-generator` path under the root directory.